feather/base/src/anvil/entity.rs

```diff
@@ -236,11 +238,19 @@ pub struct ItemNbt {
 
 impl ItemNbt {
     /// Create an `ItemStack` of the specified item and amount, setting any nbt present.
+    /// Panics if count is zero.
```

Comment:
I believe the standard for panicking in documentation is like this:

```suggestion
    /// Create an `ItemStack` of the specified item and amount, setting any NBT present.
    ///
    /// # Panics
    /// Panics if `count` is zero.
``` 

At a minimum make sure to add spacing for markdown formatting. 

![image](https://user-images.githubusercontent.com/4877041/134789151-d04a0b7f-88db-4aee-8836-86385315d6b5.png)